docs/html/cli/pip_search.rst

```diff
@@ -3,23 +3,15 @@
 ==========
 pip search
 ==========
-
-
-
-Usage
-=====
-
-.. tab:: Unix/macOS
-
-   .. pip-command-usage:: search "python -m pip"
-
-.. tab:: Windows
-
-   .. pip-command-usage:: search "py -m pip"
+.. deprecated:: 20.3.4
 
 
 Description
 ===========
+.. warning::
+    PyPI no longer supports ``pip search`` (or XML-RPC search). Please use https://pypi.org/search (via a browser)
+    instead. See https://warehouse.pypa.io/api-reference/xml-rpc.html#deprecated-methods for more information.
+    This command may still be supported by indexes other than PyPi.
```

Comment:
```suggestion

    However, XML-RPC search (and this command) may still be supported by indexes other than PyPI.
```

Let's make this a separate sentence and explicitly call out XML-RPC search. Also, we should fix the capitalisation for PyPI here. :)